pkgs/by-name/hu/hujsonfmt/package.nix

```diff
@@ -0,0 +1,30 @@
+{ lib, buildGoModule, fetchFromGitHub }:
+
+let
+  version = "0-2022-12-23";
```

Comment:
```suggestion
  version = "0-unstable-2022-12-23";
```

Reply:
Didn't give it much thought, another package I patterned this after did the same thing. I've moved it inside now.